test/Tests.Infrastructure/RavenTestBase.cs

```diff
@@ -628,7 +628,7 @@ protected async Task<T> WaitForValueAsync<T>(Func<Task<T>> act, T expectedVal, i
                 }
                 catch
                 {
-                    if (sw.ElapsedMilliseconds <= timeout)
+                    if (sw.ElapsedMilliseconds > timeout)
```

Comment:
Why? We should throw then, shouldn't we?

Reply:
I think the purpose here it not throw if the time of `timeout` not passed. 
What is the benefit of not throwing when the time passed?

Reply:
Sorry, there are two similar conditions there and I didn't notice that this one is inside `catch`.